### DISCUSSION TITLE
How to disable system wide notifications

### DISCUSSION THREAD

**Discussion by jfamousket** [LazyVim/LazyVim#3708] [Q&A] [answered]
I would like to get rid of these notifications which popup multiple times immediately I unfocus my terminal window

I am fine with them showing up in my terminal but I don't want them creeping to system notifications when the window isn't focused

![image](https://github.com/LazyVim/LazyVim/assets/40635633/afb8fca4-7613-43a2-b61f-482d173e1108)

**Comment by folke** [comment, marked as answer]
That is **NOT** default noice/LazyVim behavior.
You probably took that from my own dots.
I have noice setup in a way that will show a desktop notification when I get a notification and NEovim is not the focused window.
My code excludes LSP notifs though...

**Reply by jfamousket** [reply]
Thanks I indeed did
